app/models/project_improvement.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+# PRs (improvements) flow to the original project
+class ProjectImprovement < ApplicationRecord
+  belongs_to :project
+
+  validates :repository_url, uniqueness: true
```

Comment:
```suggestion
  validates :repository_url, uniqueness: { case_sensitive: false }
```
possible https://github.com/evrone/Evrone_Opensource
```
$ git clone git@github.com:evrone/Evrone_Opensource.git
Клонирование в «Evrone_Opensource»…
remote: Enumerating objects: 44, done.
remote: Counting objects: 100% (44/44), done.
remote: Compressing objects: 100% (31/31), done.
remote: Total 484 (delta 11), reused 41 (delta 9), pack-reused 440
Получение объектов: 100% (484/484), 196.99 KiB | 794.00 KiB/s, готово.
Определение изменений: 100% (183/183), готово.
```

Reply:
good catch, we should make `citext` as a database type as well